lib/eslint.js

```diff
@@ -46,9 +46,10 @@ estraverse.VisitorKeys.Property.push("argument");
  * Parses a list of "name:boolean_value" or/and "name" options divided by comma or
  * whitespace.
  * @param {string} string The string to parse.
+ * @param {Comment} comment The comment node which has the string.
  * @returns {Object} Result map object of names and boolean values
  */
-function parseBooleanConfig(string) {
+function parseBooleanConfig(string, comment) {
```

Comment:
I think we should add unittests to verify that comments are attached to the resulting object.